packages/cli/scripts/preinstall.js

```diff
@@ -63,6 +64,7 @@ function parsePackageManagerName(userAgent) {
   // - https://pnpm.js.org/en/3.6/only-allow-pnpm
   // - https://github.com/cameronhunter/npm-config-user-agent-parser
   if (userAgent) {
+    // eslint-disable-next-line no-useless-escape -- ???
```

Comment:
I thought I fixed that on main, is this branch up-to-date already?

Reply:
Merged with main